StartupSession/Link/Test.dyalog

```diff
@@ -1812,8 +1812,13 @@
           :If ⎕SE.Link.Watcher.DOTNET ⍝ Test #397: Inject a .NET namespace
               name⍎'⎕USING←''''' ⋄ z←name⍎'System'
           :EndIf
+          '#.ignoreme' ⎕NS ''         ⍝ Test #442: Ignore "refs"
```

Comment:
I think this should say `'#.outside' ⎕NS ''` as the doubled use of the name `ignoreme` is confusing.